test/support/fixtures.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Ebaze.Fixtures do
+  use Phoenix.ConnTest
+
+  alias Ebaze.Accounts
+
+  @endpoint EbazeWeb.Endpoint
+
+  def fixture(:invalid_attrs) do
```

Comment:
perhaps prefix by object type?

Reply:
like auction invalid attrs

Reply:
also I really like how it's using it by the pattern matching and not functions, this feels really slick